### ISSUE TITLE
[User data] Add "User data" tab to VM creation

### ISSUE THREAD

**Issue by ershovaelena** [bwsw/cloudstack-ui#1460]
This issue covers the following requirements:
[REQ_VM_CREATE_026] A user must be able to add user data in VM creation

*To provide PM with mockups*

**Comment by rennervo** [comment]
Lack: 1

### 1) There is no error handling for invalid characters for User data

An error about invalid characters does not appear when entering characters outside of the base64 range

**Steps:**
1. Open the VM creation window
2. Click the "User data" tab
3. Enter Russian characters in the "User data" field
4. Fill in the required fields
5. Click Create.

**Actual result:** Nothing happens, an error appears in the console (see screenshot)
**Expected result:** An error about incorrect characters appears in the "User data" field or the "User data" field is highlighted in red, the create button is not available for pressing
![without base64 symbols](https://user-images.githubusercontent.com/26665398/52402558-29516680-2af7-11e9-81d2-5e9a8f5dcd6a.png)

### 2) Questions for discussion with the PM:

- Vertical scroll is always present in the "User data" field, is this correct? Can add it only after a certain height of the entered value?
![scroll without value in field](https://user-images.githubusercontent.com/26665398/52402635-5f8ee600-2af7-11e9-801d-2abc932691c1.png)

- With a long vertical value, there are 2 scroll bars. Perhaps it is worth leaving only one? It seems to me better to leave only the inner one.
![two scroll-bars](https://user-images.githubusercontent.com/26665398/52402694-8816e000-2af7-11e9-887c-26d03d723a92.png)

- In browsers [Chrome, Chromium, Opera, Safari] there is a horizontal scroll when entering multiple spaces in a row. If you enter not spaces, but ordinary characters, then the string reaching the end of the line is transferred. Make the behavior to one way: or a horizontal scroll is available for any characters, or not available at all.
![horisontal scroll](https://user-images.githubusercontent.com/26665398/52402798-bf858c80-2af7-11e9-92c1-f7123c6817f5.png)

- Why did you decide not to translate into Russian the name of the tab and the fields for entering "User data". Maybe worth translating?

**Comment by rennervo** [comment]
@ershovaelena @dron8552

**Comment by dron8552** [comment]
Fixed:
- base64 encoding for non-latin symbols
- Vertical scroll bugs
Won't fix:
- Horizontal scroll when entering multiple spaces - this is the angular-material component behavior, you can reproduce it [here](https://material.angular.io/components/input/examples)
- User data shouldn't be translated to Russian, see requirements REQ_VM_CREATE_026, REQ_VM_DTLS_003, REQ_VM_DTLS_053
